src/clr-angular/data/stack-view/stack-block.ts

```diff
@@ -98,4 +110,25 @@ export class ClrStackBlock implements OnInit {
       this.expandedChange.emit(this.expanded);
     }
   }
+
+  onStackBlockFocus(focusState: boolean): void {
+    this.focused = focusState;
+  }
+
+  get caretDirection(): string {
+    return this.expanded ? 'down' : 'right';
+  }
+
+  get role(): string {
+    return this.expandable ? 'button' : null;
+  }
+
+  get tabIndex(): string {
+    return this.expandable ? '0' : null;
+  }
+
+  @HostBinding('class.on-hover')
+  get onHover(): boolean {
```

Comment:
Naming is off. I know it's the same styles as the hover one, but the actual behavior here has nothing to do with hover, it's just focus.

Reply:
agreed. fixed.